test/l.jl

```diff
@@ -101,22 +101,22 @@
             "                     30, @1996-JAN-1",
             "                     31, @1997-JUL-1",
             "                     32, @1999-JAN-1 )"]        
-            kernelFile = open(kernel, 'w') 
-            kernelFile.write('\\begindata\n')
-            for line in textbuf:
-                kernelFile.write(line + "\n")
+            kernelFile = open(kerneltemp, "w") 
+            write(kernelFile, "\\begindata\n")
+            for line in textbuf
+                write(kernelFile, line + "\n")
```

Comment:
This will not work

Reply:
This Is already fixed in the latest push